src/Aspire.Hosting.Azure.AppService/AzureAppServiceWebsiteContext.cs

```diff
@@ -306,6 +313,9 @@ static FunctionCallExpression Join(BicepExpression args, string delimeter) =>
             });
         }
 
+        // Added appsetting to identify the resource in a specific aspire environment
+        webSite.SiteConfig.AppSettings.Add(new AppServiceNameValuePair { Name = "ASPIRE_ENVIRONMENT_NAME", Value = environmentContext.Environment.Name });
```

Comment:
I’m confused at why this is needed. Can you explain?

Each app should point to its environments dashboard, right?